src/components/SearchInput/SearchInput.test.tsx

```diff
@@ -0,0 +1,187 @@
+import { render, screen } from "@testing-library/react"
+import userEvent from "@testing-library/user-event"
+import { act } from "react-dom/test-utils"
+import { SearchInput } from "./SearchInput"
+
+describe("SearchInput", () => {
```

Comment:
Looks like we're missing 2 branches (16/18):

<img width="543" alt="Screen Shot 2022-08-17 at 11 09 03 AM" src="https://user-images.githubusercontent.com/157195/185190124-d14d3fc6-daa7-4d75-b4a8-11a34ca2bf49.png">

Reply:
I love that you are calling this out! It's one of the reasons I try to avoid `if/else`s at all costs! They automatically create branching logic that is often a pain to test